frontend/tests/utils/helpers.ts

```diff
@@ -9,21 +15,30 @@ export async function goToPage(url: string = TEST_URL) {
 
 export async function login() {
   goToPage();
+  expect(process.env.TEST_ACCOUNT_PASS).toBeDefined();
   try {
     await expect(page).toHaveSelector(getText("My Collections"), {
       timeout: TIMEOUT_MS,
     });
   } catch (error) {
     const url = page.url();
-    await page.click(getText("Log In"));
-
-    await page.fill('[name="email"]', TEST_EMAIL);
-    // await page.fill('[name="password"]', password);
-
-    await Promise.all([
-      page.waitForNavigation({ waitUntil: "networkidle" }),
-      page.click('[name="submit"]'),
-    ]);
+    if (TEST_ENV === "happy") {
+      await page.click(getText("Log In"));
+      await page.fill('[name="Username"]', TEST_USERNAME);
+      await page.fill('[name="Password"]', TEST_PASSWORD);
+      await Promise.all([
+        page.waitForNavigation({ waitUntil: "networkidle" }),
+        page.click('[value="login"]'),
+      ]);
+    } else {
+      await page.click(getText("Log In"));
+      await page.fill('[name="email"]', TEST_EMAIL);
+      await page.fill('[name="password"]', TEST_PASSWORD);
+      await Promise.all([
+        page.waitForNavigation({ waitUntil: "networkidle" }),
+        page.click('[name="submit"]'),
+      ]);
+    }
```

Comment:
mock OIDC uses username, Auth0 uses email

Reply:
Hmm OIDC's login button doesn't have `name='submit'`, does `page.click('[name="submit"]'` still work?

If not, yesterday I used:

```ts
await Promise.all([
      page.waitForNavigation({ waitUntil: "networkidle" }),
      page.click(':is(button:has-text("Log In"), button:has-text("Login"))'), // <--- THIS!
    ]);
```

<img width="757" alt="Screen Shot 2021-04-02 at 10 35 45 AM" src="https://user-images.githubusercontent.com/6309723/113439466-7eb7f400-939f-11eb-98e1-ae5534df2822.png">

Reply:
For OIDC we use value=login, auth0 uses name-submit